community-membership.md

```diff
@@ -3,21 +3,20 @@
 **Note:** This document is in progress
 
 This doc outlines the various responsibilities of contributor roles in Kubernetes.  The Kubernetes
-project is subdivided into various sub-areas.  Responsibilities for many contributor roles are
-scoped by these sub-areas.
+project is subdivided into various sub-areas.  Responsibilities for most roles is scoped to these sub-areas.
```

Comment:
I think we should rename "sub-areas" to "subprojects", here and throughout this document.